user_guides/jakartaee/src/main/jbake/content/rules-wp.adoc

```diff
@@ -404,20 +404,21 @@ corresponding tests within this TCK for those additional technologies
 must also be run.
 
 EE-WP21 Compliance testing for Jakarta EE 10 Web Profile consists of running
-the Jakarta EE 10 Web Profile tests and the following Technology
-Compatibility Kits (TCKs):
-
-* Jakarta Authentication 3.0
-* Jakarta Bean Validation 3.0
-* Jakarta Concurrency 3.0
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Faces 4.0
-* Jakarta JSON Processing 2.1
-* Jakarta JSON Binding 3.0
-* Jakarta RESTful Web Services 3.1
-* Jakarta Security 3.0
-* Jakarta XML Binding 4.0 (If XML Binding is supported)
+the Jakarta EE 10 Web Profile TCK tests and the following Technology
+Compatibility Kits (TCKs). Version details are defined in the Platform EE Specification document (https://jakarta.ee/specifications/webprofile/10/), see heading 'Web Profile Definition', sub-heading 'Required Components':
+
+* Jakarta Authentication
+* Jakarta Bean Validation
+* Jakarta Concurrency
+* Jakarta Contexts and Dependency Injection
+* Jakarta Debugging Support for Other Languages
```

Comment:
Same as mentioned above, Debugging Support wasn't a required TCK before and probably shouldn't be now.  The same for Jakarta Mail + Activation.

Reply:
That was an error in previous versions of the document. The Spec. has always been a requirement and the TCK must be run.